### ISSUE TITLE
HTTPS unsupported

### ISSUE THREAD

**Issue by malsher** [hapijs/tv#126]
This is excellent and of huge value to anyone working with hapi.js

However, it does not support https as some of the content is being requested over http.

The result is reported by Chrome as follows:

```
Mixed Content: The page at 'https://www.mydomain.com/debug/console' was loaded over HTTPS, but requested an insecure stylesheet 'http://fonts.googleapis.com/css?family=PT+Mono|Open+Sans:300,400,600'. This request has been blocked; the content must be served over HTTPS
```

Although it appears to only be fonts, it is enough to stop any content being rendered.

**Comment by osukaa** [comment]
would it be enough just to update the link to https in the template?

**Comment by malsher** [comment]
Or possibly remove http: altogether and allow the browser to decide, as discussed here:

http://stackoverflow.com/questions/18026809/google-web-fonts-on-https-pages-on-chrome

**Comment by malsher** [comment]
Great that you've resolved this issue osukaa.

I can confirm that the reported issue is resolved.

There is now a new https issue (https://github.com/hapijs/tv/issues/128), but I suspect it is unrelated other than being https so created it as a seperate issue ticket.

I appreciate your fast fix on #126.
